LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2023 gittuf a Series of LF Projects, LLC
```

Comment:
My understanding is we don't need to continuously bump this, and LF recommendation is that we don't need the year at all https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects

Reply:
Interesting. I'll take the year out altogether then. cc @hythloda

Reply:
Done.